src/XrdSys/XrdSysPthread.hh

```diff
@@ -351,7 +351,7 @@ enum PrefType {prefWR=1};
 
         XrdSysRWLock(PrefType ptype)
                     {
-#ifdef __linux__
+#if defined(__linux__) && !defined(MUSL)
```

Comment:
This may actually be a problem, since it's on a header. I may use an alternative for glibc and uclibc that does not need the value of the `MUSL` macro.

Reply:
Fixed by using `__GLIBC__` and `__UCLIBC__` defines instead.